lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -2,3 +2,52 @@
 <section>
     {{!-- Fill out this registration form. --}}
 </section>
+<Registries::DraftRegistrationManager
+    @currentPage={{this.model.pageIndex}}
+    @modelTaskInstance={{this.model.taskInstance}}
+    @updateRoute={{route-action 'updateRoute'}}
+    as |draftRegistrationManager|
+>
+    <OsfLayout 
+        @backgroundClass='{{local-class 'ContentBackground'}}'
+        as |layout|>
+        <layout.leftNav as |leftNav|>
+            {{#each draftRegistrationManager.pageManagers as |pageManager index|}}
+                {{#let (hash
```

Comment:
I think adding a <leftNav.link> wrapper component might help tidy things up.

```handlebars
<PageLink @pageManager={{@pageManager}} @index={{index}} @currentPage={{currentPage}} as |pageLink|>
   <leftNav.link
     @route="registries.drafts.draft"
     @models={{pageLink.models}}
     @icon={{pageLink.icon}}
     @label={{pageManager.pageHeadingText}}
     @class={{pageLink.validationClass}}
   >
     ....
   </leftNav.link>
</PageLink>
{{!-- page-link/template.hbs --}}
{{yield (hash
     validationClass=(local-class this.validationClass)
     models=(array draftId pageIndexWithSlug)
     icon
)}}
```
Use page-param util to compute `pageIndexWithSlug`